Library/Homebrew/cmd/test.rb

```diff
@@ -10,6 +10,9 @@
 #:    If `--debug` is passed and the test fails, an interactive debugger will be
 #:    launched with access to IRB or a shell inside the temporary test directory.
 #:
+#:    If `--keep-tmp` is passed, the temporary files created for the test are
+#:    not deleted.
```

Comment:
Is it an oversight that you didn't update the documentation for `install` or are you just postponing this until the PR is basically finalized?